src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let total;
+
+  if (days < 3) {
+    total = 40 * days;
+  } else if (days >= 3 && days < 7) {
+    total = (40 * days) - 20;
+  } else {
+    total = (40 * days) - 50;
+  }
```

Comment:
You multiply days with the fee 3 times on each check. Your code will require less processing if you multiply days with the fee outside just once outside the conditions, and then deduct the discount inside the conditions.